### ISSUE TITLE
merging episode: add time estimates

### ISSUE THREAD

**Issue by chrissyrissmeyer** [ucsdlib/git-novice#64]

**Comment by dunn** [comment]
Should wait on https://github.com/ucsdlib/git-novice/issues/65#issuecomment-340847258

**Comment by chrissyrissmeyer** [comment]
this episode was merged into branching. do you think 25 minutes will be enough time for both? or should we plan on 30?

**Comment by dunn** [comment]
As it was yesterday, at least 30.  But we ought to try to reduce that.

**Comment by dunn** [comment]
I would put it as 30 to be safe, since we might have to walk around a bit to make sure everyone had a successful merge.

**Comment by chrissyrissmeyer** [comment]
https://github.com/ucsdlib/git-novice/pull/86
